tests/unit/FilingHistoryList.spec.ts

```diff
@@ -1315,7 +1316,8 @@ describe('Filing History List - alterations', () => {
             name: 'alteration',
             date: '2020-03-24T19:20:05.670859+00:00',
             status: 'COMPLETED',
-            filingId: 85114
+            filingId: 85116,
```

Comment:
This is the dumbest thing: SonarCloud was complaining about the duplicated 2 lines, so I changed the first line to work around that error 😛